Examples/Physics_applications/QED/inputs.2d

```diff
@@ -57,6 +61,10 @@ electrons.uz_m = 500.
 electrons.ux_th = 2.
 electrons.uy_th = 2.
 electrons.uz_th = 50.
+##########QED####################
+electrons.do_qed = 1
+electrons.do_qed_quantum_sync = 1
```

Comment:
IF do_qed=1 this will enable Quantum Synchrotron process for the species. 
In principle we should check if the species is either electrons or positrons, but we are not doing that now.
One question: have you already implemented classical radiation reaction somewhere or is someone working on it?

Reply:
Ok! I could do that then, since it is even more urgent than QED (I've opened a " dedicated issue")